library/alloc/src/rc.rs

```diff
@@ -951,11 +1398,14 @@ impl<T: ?Sized> Rc<T> {
     #[must_use = "this returns a new `Weak` pointer, \
                   without modifying the original `Rc`"]
     #[stable(feature = "rc_weak", since = "1.4.0")]
```

Comment:
```suggestion
    #[inline]
    #[must_use = "this returns a new `Weak` pointer, \
                  without modifying the original `Rc`"]
    #[stable(feature = "rc_weak", since = "1.4.0")]
```

Reply:
Seems weird to add `#[inline]` to a method I didn't add?

Reply:
I think the contents of `downgrade` changed right? A few of the suggestions I sent add inlines in places where you didn't touch much but but possibly have downstream connections to your stuff, or might have a chance of boosting the perf run which isn't bad (since it's evaluating the entire Arc/Rc module and not just your stuff)

But I'm neither a team member nor an expert, so feel free to use your discretion of course

Reply:
Responded to the comment above, I think inlining could help but needs to be tested. good spot!